.golangci.yml

```diff
@@ -2,6 +2,7 @@
 run:
   concurrency: 6
   deadline: 5m
+  modules-download-mode: vendor
```

Comment:
This should not be needed, do we have to re-vendor with go 1.13 to fix this up?

Reply:
I am not sure the issue. I can try that.

Reply:
ah nice that worked